requirements/edx/github.txt

```diff
@@ -97,4 +97,4 @@ git+https://github.com/edx/edx-proctoring.git@0.13.0#egg=edx-proctoring==0.13.0
 # Third Party XBlocks
 -e git+https://github.com/mitodl/edx-sga@172a90fd2738f8142c10478356b2d9ed3e55334a#egg=edx-sga
 -e git+https://github.com/open-craft/xblock-poll@v1.1#egg=xblock-poll==1.1
-git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.0.8#egg=xblock-drag-and-drop-v2==2.0.8
+git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.0.9#egg=xblock-drag-and-drop-v2==2.0.8
```

Comment:
The version number occurs twice on this line.  Please change the secon occurrence as well.